Sources/Commands/Error.swift

```diff
@@ -30,7 +30,7 @@ extension Error: CustomStringConvertible {
         case .invalidToolchain(let problem):
             return problem
         case .rootManifestFileNotFound:
-            return "root manifest not found"
+            return "Could not find \(Manifest.filename) in this folder or any of it's parent folders."
```

Comment:
Thanks for the fix here!  I didn't actually realize that SwiftPM would look up the parent folder hierarchy, but that makes sense, since you could invoke it in a subdirectory.

I'd suggest using "directory" rather than "folder" though, since that's what the `swift build --help` terminology does and "folder" is Mac-specific.

```suggestion
            return "Could not find \(Manifest.filename) in this directory or any of its parent directories."
```